examples/observability/README.md

```diff
@@ -0,0 +1,69 @@
+Also, please ensure that the Prometheus Operator has the required permissions to watch resources of the `keptn-lifecycle-controller-system` namespace (see https://prometheus-operator.dev/docs/kube/monitoring-other-namespaces/ as a reference).
+For setting up both Jaeger and Prometheus, please refer to their docs:
+
+- [Jaeger Setup](https://github.com/jaegertracing/jaeger-operator) 
+- [Prometheus Operator Setup](https://github.com/prometheus-operator/kube-prometheus/blob/main/docs/customizing.md)
+
+In this tutorial, we will assume that Jaeger will be installed in the `keptn-lifecycle-controller-system`, and the Jaeger collector endpoint is reachable under `http://jaeger-collector:14250`.
+
+## Configuring the OpenTelemetry Collector and Prometheus ServiceMonitor
+
+Once Jaeger and Prometheus are installed, you can deploy and configure the OpenTelemetry collector using the manifests in the `config` directory:
+
+```sh 
+kubectl apply -f config/
+```
+
+Also, please ensure that the `OTEL_COLLECTOR_URL` env vars of both the `klc-controller-manager`, 
+as well as the `keptn-scheduler` deployments are set appropriately. 
+By default, they are set to `otel-collector:4317`, which should be the correct value for this tutorial.
+
+Eventually, there should be a pod for the `otel-collector` deployment up and running:
+
+```sh
+$ kubectl get pods -lapp=opentelemetry -n keptn-lifecycle-controller-system
+
+NAME                              READY   STATUS    RESTARTS      AGE
+otel-collector-6fc4cc84d6-7hnvp   1/1     Running   6 (51m ago)   92m
+```
+
+## Seeing the OpenTelemetry Collector in action
+
+After everything has been set up, use the lifecycle operator to deploy a workload (e.g. using the `single-service` or `podtato-head` example in the `examples` folder).
+Once either of these examples have been deployed, you can view the generated traces in Jaeger. To do so, please create a port-forward for the `jaeger-query` service:
+
+```sh
+kubectl port-forward -n keptn-lifecycle-controller-system svc/jaeger-query 16686 
+```
+
+Afterwards, you can view the Jaeger UI in the browser at [](http://localhost:16686). There you should see the traces generated by the lifecycle controller, which should look like this:
+
+**Traces overview** 
+
+![](./assets/traces_overview.png)
+
+**Trace details**
+
+![](./assets/trace_detail.png)
+
+In Prometheus, do a port forward to the prometheus service inside your cluster (the exact name and namespace of the prometheus service will depend on your Prometheus setup - we are using the defaults that come with the example of the Prometheus Operator tutorial).
+
+```sh
+kubectl -n monitoring port-forward svc/prometheus-k8s 9090
+```
+
+Afterwards, you can view the Prometheus UI in the browser at [](http://localhost:9090). There, in the [Targets](http://localhost:9090/targets?search=) section, you should see an entry for the otel-collector:
```

Comment:
there is nothing in the  clickable part of the link, this won't be seen , change to [loclahost](http://localhost:9090)

Reply:
good catch, fixed